bentoml/deployment/aws_lambda/utils.py

```diff
@@ -31,7 +31,10 @@
 
 def ensure_sam_available_or_raise():
     try:
+        # pylint: disable=unused-import
         import samcli  # noqa
+
+        # pylint: enable=unused-import
```

Comment:
does this work? ```import samcli  # pylint: disable=unused-import```

Reply:
actually might be nice to add a minimal required version check here, something like:
```
assert list(map(int, samcli.__version__.split('.'))) > [0, 31, 1], "Upgrade sam cli with pip install -U ............."
```